ski_conditions/settings.py

```diff
@@ -35,6 +35,7 @@
 ]
 
 LOCAL_APPS = [
+    'app_scraping.apps.AppScrapingConfig',
```

Comment:
Move this into `ski_conditions.apps.scraping`.

Reply:
Done in commit 410f8d1